src/plugins/auto/README.md

```diff
@@ -0,0 +1,51 @@
+# OpenVINO™ AUTO Plugin
+
+The main responsibility of the AUTO plugin is to provide a unified device that enables developers to code deep learning applications once and deploy them anywhere. 
+
+Other capabilities of the AUTO plugin include:
+* Static device selection, which intelligently loads a network to one device or multiple devices.
+* CPU acceleration to start inferencing while the target device is still loading the network.
+* Model priority support for loading multiple networks to multiple devices.
+
+The component is written in `C++`. If you want to contribute to the AUTO plugin, follow [the common coding style rules](../../../docs/dev/coding_style.md).
+
+## Key contacts
+
+In case of any questions, review and merge requests, contact the [AUTO Plugin maintainer group](https://github.com/orgs/openvinotoolkit/teams/openvino-ie-auto-multi-maintainers).
+
+## Components
+
+The AUTO plugin follows the OpenVINO™ plugin architecture and consists of several main components:
+ * [docs](./docs) contains developer documentation pages for AUTO plugin.
+ * [src](./) folder contains sources of the core component.
```

Comment:
I see only one folder. Plugin doesn't have `src` folder.
I propose to move all sources to src folder.
Or maybe:
```suggestion
 * [current](./) folder contains sources of the core component.
```
But I prefer option 1

Reply:
Update folder name as option 2 so far.

Create ticket 100839 to move all sources to src folder when this change has less impact on others.